EventHubs/src/EventHubs_Logs/lib/datatransformer.js

```diff
@@ -67,30 +65,57 @@ Transformer.prototype.getProperties = function (resourceGroupName) {
  * @returns {object} of log following GCM's standard format
  */
 Transformer.prototype.generateFormattedLog = function (context, msg) {
-    var properties = this.getProperties(this.getValue(() => msg.properties.resource.resourceGroupName, ''));
+    // Logs from Azure Function follow different format than logs from Logic App
+    if(msg.category == 'FunctionAppLogs') {
```

Comment:
which function app logs is this getting?

Reply:
This can get any diagnostic logs that log to the event hub from the function. In our case, we are using PowerShell, so we are capturing the Write-Warning/Error logs from the function during runtime. Ill update comment for clarification

Reply:
is it any non-dotnet function or all functions?

@jnevins-gcm should probably make sure our python functions can log to sumo, right?

Reply:
As far as I know, you can turn the diagnostic settings on for any function.

Reply:
@swettstein  yes we should